src/_tutorials/web/low-level-html/connect-dart-html.md

```diff
@@ -431,21 +459,21 @@ a class, or an attribute.
 Selectors can also be nested.
 
 CSS selectors are important in Dart programs
-because you use them with querySelector() and querySelectorAll()
+because you use them with `querySelector()` and `querySelectorAll()`
 to get matching elements from the DOM.
-Most often Dart programs use ID selectors with querySelector()
-and class selectors with querySelectorAll().
+Most often Dart programs use ID selectors with `querySelector()`
+and class selectors with `querySelectorAll()`.
 
 Here are some examples of CSS selectors:
 
 | Selector type | Example | Description |
 |---|---|
-| ID selector | #RipVanWinkle | Matches a single, unique element |
-| HTML element | p | Matches all paragraphs |
-| HTML element | h1 | Matches all level-one headers |
-| Class | _.classname_ | Matches all items with the class _classname_  |
-| Asterisk | * | Matches all elements |
-| Attribute | input[type="button"] | Matches all button input elements |
+| ID selector | `#RipVanWinkle` | Matches a single, unique element |
+| HTML element | `p` | Matches all paragraphs |
+| HTML element | `h1` | Matches all level-one headers |
+| Class | `_.classname_` | Matches all items with the class _classname_  |
```

Comment:
_.classname_ isn't rendering correctly in the second column. Maybe use a non-code font for everything after the period? The following may or may not work ;)
```suggestion
| Class | <code>.<em>classname</em></code> | Matches all items with the class _classname_  |
```